app/code/BVMyBlog/Blog/view/frontend/web/js/blog-back-url.js

```diff
@@ -0,0 +1,23 @@
+define([
```

Comment:
Why do you need this widget? Use **mage/redirect-url**. Just specify **url** argument
```
<button class="blog-back-url"
            data-mage-init='{"redirectUrl":{"url": "<?= $block->escapeJS($block->getBackUrl()) ?>"}}'
            type="button"
            title="Back to product"
            id="backToProduct">
        <span>Back to product</span>
    </button>
```